tools/crm_resource_runtime.c

```diff
@@ -134,12 +134,13 @@ find_resource_attr(pcmk__output_t *out, cib_t * the_cib, const char *attr,
         xmlNode *child = NULL;
 
         rc = EINVAL;
-        printf("Multiple attributes match name=%s\n", attr_name);
+        out->info(out, "Multiple attributes match name=%s", attr_name);
+        out->spacer(out);
```

Comment:
Before this PR, the spacer was after the values rather than before

Reply:
Fixed.